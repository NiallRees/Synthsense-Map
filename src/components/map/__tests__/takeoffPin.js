import TakeoffPin from '../takeoffPin';
import React from 'react';
var enzyme = require('enzyme');

describe('Test MouseCoords', () => {
  it('Renders', () => {
    const component = enzyme.shallow(<TakeoffPin/>);
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  })
})