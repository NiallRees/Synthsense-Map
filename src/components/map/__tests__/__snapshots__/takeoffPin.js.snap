// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test MouseCoords Renders 1`] = `ShallowWrapper {}`;
